src/fragments/lib/datastore/flutter/data-access/observe-update-snippet.mdx

```diff
@@ -0,0 +1,47 @@
+```dart
+class MyApp extends StatefulWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  StreamSubscription<QuerySnapshot<Post>>? _stream;
+
+  // A reference to the retrieved post
+  late Post _post;
+
+  // Initialize the Amplify libraries
```

Comment:
It may not be immediately obvious where this should happen and where `observeQuery` should be called.

```suggestion
  @override
  void initState() {
    super.initState();
    _configure();
  }
  
  // Initialize the Amplify libraries and call `observeQuery`
  Future<void> _configure() async {
    // ...
  }
```

Reply:
Thanks!